### DISCUSSION TITLE
LoRA Training?

### DISCUSSION THREAD

**Discussion by AIShank** [TheLastBen/fast-stable-diffusion#1501] [Q&A]
Was wondering if there are any methods to training LoRAs on the google colab, can't seem to find any information on them.

**Comment by gilroff** [comment]
im curious about this too

**Comment by Excalibro1** [comment]
The only way through Ben's colab is to install d8hazards dreambooth extension and train a LoRA through that otherwise best bet is the notebook linked from this LoRA repo.

https://github.com/cloneofsimo/lora
